docs/full-node/loading-snapshot.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 35
+title: Loading from a snapshot
+sidebar_label: Loading from snapshot
+---
+
+import UntranslatedPageText from "@site/src/components/UntranslatedPageText";
+
+<UntranslatedPageText />
+
+# Full node
+The very first run of a full node, as described in the [getting started](./getting-started) takes few hours until 
+it is fully in sync with the other nodes in the network, or said differently, until it downloads a copy of the entire blockchain.
+
+In order to speed up this initial sync process, snapshots are available in the
+[dedicated Alephium Archives repository](https://archives.alephium.org). An automated process upload snapshots
+for both [testnet](https://archives.alephium.org/#testnet/) and [mainnet](https://archives.alephium.org/#mainnet/) networks.
+
+When preparing the full node for its first run, just before launching it, you can download the latest snapshot and
+extract (it's a `tar` file) it at the right location. The snippet below even does this inline, i.e. without doubling the
+required disk space to download and to extract the snapshot:
+
+```shell
+ALEPHIUM_HOME=~/.alephium
+ALEPHIUM_NETWORK=mainnet
+curl -L "$(curl -s https://s3.eu-central-1.amazonaws.com/archives.alephium.org/archives/$ALEPHIUM_NETWORK/full-node-data/_latest.txt)" | tar xf - -C "$ALEPHIUM_HOME/"
```

Comment:
Will this empty the existing folder `$ALEPHIUM_HOME/$ALEPHIUM_NETWORK` and put only the downloaded data there?

Reply:
This will overwrite existing files with the one in the snapshot, leaving the new files (if any) untouched. Even if the `.sst` files are immutable, if you run the command on a running node, you'll endup with an inconsistent state. 

This command should really be used once at first run of the full node. Or drop the state folder if you want to restart. In [touilleio/alephium-standalone](https://github.com/touilleio/alephium-standalone) referenced here, I'm un'taring in a temporary folder and then using an atomic `mv` of the folder if the final one does not exist already to ensure consistency. See https://github.com/touilleio/alephium-standalone/blob/main/snapshot-loader-entrypoint-wrapper.sh#L42 and https://github.com/touilleio/alephium-standalone/blob/main/snapshot-loader-entrypoint-wrapper.sh#L59 for details.

Reply:
Nitpicking: maybe we should first remove the existing folder if there is any and then download the new copy. Sometimes, users are not aware of the existence of existing files.

Reply:
Removing the existing folder means re-downloading the snapshot at every restart of the container full-node. This is very inefficient.

Reply:
Sure, I only meant when a download is necessary. Specifically:
1. the blockchain data is synced rightly: no snapshot download, no removal
2. the blockchain data is synced but corrupted: removing the existing data and new snapshot download
  - This is case I was talking. A bit of cleanup can be helpful.
3. the blockchain data is empty: just a new snapshot download.

Reply:
Do you think it would worth mentionning that the snippet is an example only and doesn't cover all the failures, but using the wrapper explicitely developped for that is recommended?

Reply:
I am afraid people don't usually read docs 😅. 

Maybe we have one-liner CLI to basically download the [snapshot-loader-entrypoint-wrapper.sh](https://github.com/touilleio/alephium-standalone/blob/main/snapshot-loader-entrypoint-wrapper.sh) and execute the script. Something like:
```
curl -0- https://github.com/touilleio/alephium-standalone/blob/main/snapshot-loader-entrypoint-wrapper.sh | bash 
```

Reply:
Should be a declarative sentence: people don't read docs. :)

Using the wrapper script as one liner is a very good idea! I'll update.

Reply:
I had to split the script in https://github.com/touilleio/alephium-standalone to provide the sole functionality to download and extract a snapshot, in https://github.com/touilleio/alephium-standalone/blob/main/snapshot-loader.sh.
I updated the PR accordingly,